src/main/build/debug.c

```diff
@@ -47,8 +47,8 @@ const char * const debugModeNames[DEBUG_COUNT] = {
     [DEBUG_RX_FRSKY_SPI] = "RX_FRSKY_SPI",
     [DEBUG_RX_SFHSS_SPI] = "RX_SFHSS_SPI",
     [DEBUG_GYRO_RAW] = "GYRO_RAW",
-    [DEBUG_DUAL_GYRO_RAW] = "DUAL_GYRO_RAW",
-    [DEBUG_DUAL_GYRO_DIFF] = "DUAL_GYRO_DIFF",
+    [DEBUG_MULTI_GYRO_RAW] = "DUAL_GYRO_RAW",
+    [DEBUG_MULTI_GYRO_DIFF] = "DUAL_GYRO_DIFF",
```

Comment:
Should we go all the way (needs configurator / blackbox changes) ?
```suggestion
    [DEBUG_MULTI_GYRO_RAW] = "MULTI_GYRO_RAW",
    [DEBUG_MULTI_GYRO_DIFF] = "MULTI_GYRO_DIFF",
```